### DISCUSSION TITLE
Webhook

### DISCUSSION THREAD

**Discussion by StratISaurus** [rustdesk/rustdesk-server-pro#219] [Ideas]
With the new custom Client Generator, it would be handy to have a customizable webhook that could fire a simple get request when the application is launched. It would give a lot of flexibility for other parties to integrate it with their tools (like our help desk and ticketing system) as we could just have the Rust Desk client ping our ticket creation end point with it's ID when a user launches it.

**Comment by 21pages** [comment]
Audit webhook
